HabitTracker/InputValidator.cs

```diff
@@ -0,0 +1,153 @@
+﻿using HabitTracker;
+
+internal static class InputValidator
```

Comment:
A lot of this class' methods seem to be more than input validators. They seem to be threefold:
- Prompt user to give input
- Take input
- validate input

It would be a good exercise to try to separate some of these things out into separate methods.